pkg/apis/serving/v1beta1/revision_types.go

```diff
@@ -85,7 +84,7 @@ type RevisionSpec struct {
 	// requests per container of the Revision.  Defaults to `0` which means
 	// unlimited concurrency.
 	// +optional
-	ContainerConcurrency RevisionContainerConcurrencyType `json:"containerConcurrency,omitempty"`
+	ContainerConcurrency av1alpha1.AutoscalerContainerConcurrencyType `json:"containerConcurrency,omitempty"`
```

Comment:
It's cosmetic, but I'm not thrilled at the optics of having our beta types have alpha dependencies.

Reply:
We could bump promote autoscaling to beta? 😄

Reply:
I think having this type at all is kinda dumb.  Can we just make the transition to `*int` happen?

On another note, I was thinking that we should add something like `PropagateChildStatus` to `kmeta.OwnerRefable` for use in places like [this](https://github.com/knative/serving/pull/4969/files#diff-590ffc8d5597e284aba08273a5febed0R79).

You'd need something like: `PropagateChildStatus(runtime.Object)` and a type switch, but I think it would be a useful callback to have a generic form of.

Reply:
> I think having this type at all is kinda dumb. Can we just make the transition to *int happen?

Yes, but it has `Validate` defined on it (and used in a bunch of places). I'll move that around.

Reply:
@mattmoor it's currently `int64`, you want it to be `*int`?

Reply:
I believe their is another PR with that in-flight, so you should certainly coordinate with that.  I'm after the pointer, the bit width doesn't really matter since we cap it at 1k.